src/main/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfig.java

```diff
@@ -142,18 +142,20 @@ public FormValidation doCheckContainer(@QueryParameter String container){
     }
 
     public FormValidation doCheckPrefix(@QueryParameter String prefix){
-        FormValidation ret = FormValidation.ok();
-        if (StringUtils.isBlank(prefix)){
-            ret = FormValidation.warning("Prefix is needed to archive files in a subdirectory.");
-        } else if (!prefix.endsWith("/")){
-            ret = FormValidation.warning("if Prefix point to a folder, it should end with '/' character");
+        FormValidation ret;
+        if (StringUtils.isBlank(prefix)) {
+            ret = FormValidation.ok("BTW you might want a prefix etc.");
```

Comment:
I did not mean for that literal text to be used! :-) Was just sketching the control flow to clarify my comments.

Reply:
noted, do not C&P without reading what tell the text 🤦‍♂️